lib/farscape/transition.rb

```diff
@@ -48,6 +48,13 @@ def invoke(args = {})
         # This makes it impossible to use URIs with parameters.
         call_options[:url] = @transition.uri
         call_options[:body] = (call_options[:body] || {}).merge(params) if params.present?
+
+        if call_options[:headers][:Accept] =~ /\Aapplication\/.*json\z/ &&
+          call_options[:body] && call_options[:body].respond_to?(:to_json)
```

Comment:
- btw what's up with doing `[:Accept]` and then setting the content type using a string instead of a symbol below? if both are fine I think it should be ["Accept"] (symbols with uppercase chars are kind of weird)